src/API/WesternStatesWater.WestDaat.Common/DataContracts/ApplicationSupportingDocumentExistsResponse.cs

```diff
@@ -0,0 +1,10 @@
+namespace WesternStatesWater.WestDaat.Common.DataContracts;
+
+public class ApplicationSupportingDocumentExistsResponse : ApplicationLoadResponseBase
+{
+    public bool DocumentExists { get; set; }
+    
+    public Guid? ApplicantId { get; set; }
```

Comment:
It's probably already clear but we might use `ApplicantUserId` other places
```suggestion
    public Guid? ApplicantUserId { get; set; }
```